test/views/DeclarationDetailsViewSpec.scala

```diff
@@ -557,6 +557,20 @@ class DeclarationDetailsViewSpec extends UnitViewSpec with GivenWhenThen with In
       }
     }
 
+    "display 'Fix and resubmit' button and 'Cancel' link when latest notification is rejected Amendment" in {
+      val notificationSummariesAmendmentRejected = List(dmsrejNotification, dmsdocNotification, dmsctlNotification, acceptedNotification)
+      val view = page(createSubmissionWith(notificationSummariesAmendmentRejected, AmendmentRequest))(verifiedEmailRequest(), messages)
+      val buttonGroup = view.getElementsByClass("govuk-button-group")
+      val button = buttonGroup.get(0).child(0).text()
+      val link = buttonGroup.get(0).child(1).text()
+
```

Comment:
Ouch... this is wrong. `button` and `link` have `String` type, they are not Html `Element`. Code here should be:
```
      val button = buttonGroup.get(0).child(0)
      val link = buttonGroup.get(0).child(1)

      button.text() mustBe messages("declaration.details.fix.resubmit.button")
      button.hasClass("govuk-button")

      link.text() mustBe messages("declaration.details.cancel.amendment")
      link.hasClass("gov-link")
```